src/main/java/com/uid2/shared/Const.java

```diff
@@ -74,6 +74,9 @@ public static class Config {
         public static final String EnforceJwtProp = "enforceJwt";
         public static final String MaaServerBaseUrlProp ="maa_server_base_url";
         public static final String SaltsExpiredShutdownHours = "salts_expired_shutdown_hours";
+        public static final String S3keysMetadataPathProp =  "s3_keys_metadata_path";
+
```

Comment:
Remove extra whitespace please. Also please put these with the other metadata props